src/Tribe/Commerce/PayPal/Order.php

```diff
@@ -303,25 +312,26 @@ public function hydrate_from_transaction_data( array $transaction_data ) {
 	 *
 	 * @since 4.7
 	 *
-	 * @param array $criteria  {
+	 * @param array      $args   {
 	 *                         Optional. Arguments to retrieve orders. See WP_Query::parse_query() for all
 	 *                         available arguments.
 	 *
 	 * @type int    $post_id   ID, or array of IDs, of the post(s) Orders should be related to.
 	 * @type int    $ticket_id ID, or array of IDs, of the ticket(s) Orders should be related to.
 	 * }
+	 * @param null|array $fields List of fields to hydrate, or null for all.
 	 *
 	 * @return Tribe__Tickets__Commerce__PayPal__Order[]
 	 */
-	public static function find_by( array $args = array() ) {
-		$args = wp_parse_args( $args, array(
+	public static function find_by( array $args = [], $fields = null ) {
+		$args = wp_parse_args( $args, [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: c65921623a06d13c6a9036354ae28e3a</sub>